src/web/components/SignInGate/gateDesigns/SignInGateMain.tsx

```diff
@@ -129,10 +129,18 @@ const firstParagraphOverlay = (isComment: boolean) => css`
 // This css hides all the elements in the article after the #sign-in-gate
 // using the General sibling combinator https://developer.mozilla.org/en-US/docs/Web/CSS/General_sibling_combinator
 const hideElementsCss = `
+    .article-body-commercial-selector > * {
+        display: none;
+    }
+
+    #sign-in-gate, .article-body-commercial-selector p:nth-of-type(-n+3) {
```

Comment:
Would you be able to comment this as not immediately clear :)

Reply:
i've added in comments above the variable